### PR TITLE
Increase ScriptProcessorNode.bufferLength default value to 16384

### PR DESCRIPTION
**Pull request by adambailey-**
Here are two examples using Zello Dispatch Hub application. In both examples, I am scrolling through the "Direct Contacts" list, with roughly 500 contacts.

[Archive.zip](https://github.com/zelloptt/zello-channel-api/files/8035826/Archive.zip)

### CODE REVIEW

File: sdks/js/src/classes/recorder.js [hunk 1]
```diff
@@ -11,7 +11,11 @@ class Recorder {
       throw new Error("Recording is not supported in this browser");
     }
     this.options = Object.assign({
-      bufferLength: 4096,
+      // This is a tradeoff between latency and quality. If the buffer length is too low compared
+      // to CPU usage, the audio is robotic and glitched. A maximum value for this is a reasonable tradeoff
+      // to dramatically improve quality while under moderate stress. Do not change this default.
```
Comment (vocoded) [inline sdks/js/src/classes/recorder.js:16]: i don't think we have to be quite this prescriptive, there may be circumstances where changing the default could be advisable. btw, did you happen to run tests where the length wasn't set at all?

Comment (adambailey-) [reply sdks/js/src/classes/recorder.js:16]: I did. It was extremely rough during scroll test, not acceptable for DH

Comment (vocoded) [inline sdks/js/src/classes/recorder.js:16]:
```suggestion
      // to dramatically improve quality while under moderate CPU load. Overriding is not recommended.
```

**Review by vocoded — COMMENTED** [review]
Do you know why all the distribution files changed? Were there other commits since the last build?

**Comment by adambailey-** [comment]
Gonna close this in favor of a different PR soon(tm)

**Pull request closed without merging**